### ISSUE TITLE
Unable to reproduce paper results

### ISSUE THREAD

**Issue by Mael-zys** [NVlabs/GENMO#11]
Hi, thank you for sharing this great work!

I am trying to reproduce the results in the paper. However, I am currently unable to match the reported performance. For example, here is the last row is my reproduced results on Rich dataset using the pertained model.

<img width="792" height="221" alt="Image" src="https://github.com/user-attachments/assets/6a554cf7-a0bc-4b3e-9965-77a505c8c45e" />

Could you provide more guidance on the inference commands and data preprocessing? Thank you!

**Comment by ManiacWallnut** [comment]
Hi @Mael-zys , I'm also trying to reproduce the results. Can you share your current inference commands? Now I can only run the text2motion part with the guidance of another issue under this repo.

**Comment by Mael-zys** [comment]
Hi, you can try this command. I also modify the dataset script by removing VIMO label part, because it's not used for inference.

```bash
python scripts/train.py     exp=genmo_lg     task=test     pl_trainer.devices=1     seed=42     use_wandb=true exp_name_var=test_rich logger.name=test_rich num_test_data=999999 train_datasets=[3dpw_v1] test_datasets=[rich_test] '+data.dataset_opts.test=${test_datasets}' +data.loader_opts.test.batch_size=1 +data.loader_opts.test.num_workers=1 ckpt_path=checkpoints/s050000.ckpt ~callbacks.vis.vis_text ~callbacks.metric.metric_emdb1 ~callbacks.metric.metric_emdb2 ~callbacks.metric.metric_3dpw ~callbacks.metric.metric_3dpw_occ
```

**Comment by Strive21** [comment]
Hi， I'm also trying to reproduce the results of 3DPW.But the results dont match the paper's performance.I run this command:`python scripts/train.py exp=gem_smpl global/task=test_3dpw ckpt_path=inputs/pretrained/gem_smpl.ckpt use_wandb=false`
